pom.xml

```diff
@@ -18,48 +18,42 @@
     limitations under the License.
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
         <groupId>net.openhft</groupId>
         <artifactId>java-parent-pom</artifactId>
-        <version>1.25.4</version>
-        <relativePath />
+        <version>1.26.0-SNAPSHOT</version>
+        <relativePath/>
     </parent>
 
     <artifactId>chronicle-bytes</artifactId>
-    <version>2.25ea10-SNAPSHOT</version>
+    <version>2.26ea0-SNAPSHOT</version>
     <name>OpenHFT/Chronicle Bytes</name>
     <description>Chronicle-Bytes</description>
     <packaging>bundle</packaging>
 
     <properties>
         <zero.cost.assertions>disabled</zero.cost.assertions>
-        <compiler.argument>-Xlint:deprecation</compiler.argument>
         <sonar.organization>openhft</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
-        <!-- The parent POM defines this plugin and activates it in a profile. Unfortunately this plugin will generate
-        code at @NotNull call-sites that throws IllegalArgumentException rather than NullPointerException. This will
-        break various core tests. Chronicle have raised this issue before with the maintainers of the plugin. See:
-        https://github.com/osundblad/intellij-annotations-instrumenter-maven-plugin/issues/53
-        -->
-        <se.eris.notnull.instrument>false</se.eris.notnull.instrument>
```

Comment:
safe to remove?

Reply:
This disables a plugin that has now been removed from the parent.